builder/dockerfile/builder.go

```diff
@@ -117,8 +117,7 @@ type Builder struct {
 	Aux    *streamformatter.AuxFormatter
 	Output io.Writer
 
-	docker    builder.Backend
-	clientCtx context.Context
+	docker builder.Backend
```

Comment:
This one wasn't new (added in https://github.com/moby/moby/commit/c44e7a3e632c3ea961cb8c12ba45371f54e6699c), so we can keep it separate.